spring-rabbit-junit/src/main/java/org/springframework/amqp/rabbit/junit/BrokerRunning.java

```diff
@@ -192,6 +241,105 @@ public void setHostName(String hostName) {
 		this.hostName = hostName;
 	}
 
+	/**
+	 * Set the user for the amqp connection default "guest".
+	 * @param user the user.
+	 * @since 1.7.2
```

Comment:
Well, OK: this change is for back porting any way.
So, test must be fixed appropriately.